src/Microsoft.VisualStudio.Threading.Analyzers/build/AdditionalFiles/vs-threading.MainThreadAssertingMethods.txt

```diff
@@ -0,0 +1,3 @@
+﻿Microsoft.VisualStudio.Shell.ThreadHelper.ThrowIfNotOnUIThread
```

Comment:
This is not really related, but I just realized how poorly these are named - it'll be confusing in the future if the "main" thread and the "UI" thread are different threads.

Reply:
Yup 🙄